src/devices/bus/epson_qx/sound_card.cpp

```diff
@@ -0,0 +1,122 @@
+// license:BSD-3-Clause
+// copyright-holders:Brian Johnson
+/*******************************************************************
+ *
+ * YM2149 based sound card with header for an external MPU401
+ *
+ * Board Design: https://github.com/brijohn/qx10/tree/master/kicad/ym2149_sound_card
+ *
+ *******************************************************************/
+
+#include "emu.h"
+#include "speaker.h"
+#include "sound_card.h"
```

Comment:
Please get into the habit of putting the translation unit header immediately after the prefix header, and separating different libraries’ headers with a blank line.  The MAME convention for this would be:
```
#include "emu.h"
#include "sound_card.h"

#include "speaker.h"
```